virttest/guest_agent.py

```diff
@@ -509,8 +509,7 @@ def shutdown(self, mode=SHUTDOWN_MODE_POWERDOWN):
         if mode in [self.SHUTDOWN_MODE_POWERDOWN, self.SHUTDOWN_MODE_REBOOT,
                     self.SHUTDOWN_MODE_HALT]:
             args = {"mode": mode}
-        self.cmd(cmd=cmd, args=args, success_resp=False)
-        return True
+        self.cmd(cmd=cmd, args=args)
```

Comment:
Just realized that it should ignore the error that there is no response after sending the command, so you should catch `VAgentProtocolError` and ignore it at here.

Reply:
@luckyh  You are right,  I have updated.